### DISCUSSION TITLE
Plugin Development for Pretix SaaS

### DISCUSSION THREAD

**Discussion by naveed-strativ** [pretix/pretix#4594] [Q&A] [answered]
Hello there,
I saw that the Marketplace only supports third-party apps for **Self-Hosted instances**. Is there anyway for **Pretix SaaS** would allow **any custom or third party plugins** for the Saas Platform? Or does Pretix SaaS only allow plugins developed by their own team?

**Comment by raphaelm** [comment, marked as answer]
Hi,

pretix hosted only allows plugins created by us, or at least reviewed and maintained by us. Everything else could compromise the security and stability of the entire platform.

**Reply by naveed-strativ** [reply]
Thank you very much for the clarification.
